community/sway/etc/skel/.config/waybar/config

```diff
@@ -15,6 +15,7 @@
 
     // If height property would be not present, it'd be calculated dynamically
     "height": 30,
+    "position": "bottom",
```

Comment:
added back the position since without it, it defaulted to "top". IMO the sway bar.position has no effect when using waybar.